mypy/checker.py

```diff
@@ -1647,6 +1647,24 @@ def visit_class_def(self, defn: ClassDef) -> None:
                 with self.scope.push_class(defn.info):
                     self.accept(defn.defs)
             self.binder = old_binder
+            for base in typ.mro[1:]:
+                if base.name() != 'object' and base.defn.info:
+                    for method_name, method_symbol_node in base.defn.info.names.items():
+                        if method_name == '__init_subclass__':
+                            name_expr = NameExpr(defn.name)
+                            name_expr.node = base
+                            callee = MemberExpr(name_expr, '__init_subclass__')
+                            args = list(defn.keywords.values())
+                            arg_names = list(defn.keywords.keys())  # type: List[Optional[str]]
+                            arg_kinds = [ARG_NAMED] * len(args)
+                            call_expr = CallExpr(callee, args, arg_kinds, arg_names)
+                            call_expr.line = defn.line
+                            call_expr.end_line = defn.end_line
+                            self.expr_checker.accept(call_expr,
+                                                     allow_none_return=True,
+                                                     always_allow_any=True)
+                            break
```

Comment:
Could you please add a short comment above `break` explaining we only check the first one found? Just to make it visually more prominent.